### PR TITLE
Feat/125/manager forms refactor

### PR DESCRIPTION
**Pull request by martinrobled0**
# What does this PR do?

Refactor for labels, statuses and Innovation tiers using new components of material

# Where should the reviewer start?

You have to change your role to admin

# How should this be manually tested?

- Change your role to admin
- Click on your name and select the manager option
- Create a new label/ new status/ Innovation tiers
- Try to create without complete all inputs
- Delete the new ones
- Try to merge some of them

# What are the relevant tickets?
#125 

# Screenshots
image

# Checklist

- [x] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

File: app/routes/manager/filter-tags/innovation-tiers.tsx [hunk 1]
```diff
@@ -173,38 +147,12 @@ export const action: ActionFunction = async ({ request }) => {
...
-};
-
-const InnovationTiersGrid = () => {
+function InnovationTiersGrid(){
   const fetcher = useFetcher();
   const { innovationTiers } = useLoaderData() as LoaderData;
   const createButtonText = "Create New Innovation Tier";
   const [openDeleteModal, setOpenDeleteModal] = useState<boolean>(false);
+  const [editMode, setEditMode] = useState<boolean>(false);
   const [error, setError] = useState<string>("");
```
Comment (JCoronaWize) [inline app/routes/manager/filter-tags/innovation-tiers.tsx:156]: we could remove this useState if we are not going to use it to trigger any action or display a message

File: app/routes/manager/filter-tags/statuses.tsx [hunk 2]
```diff
@@ -162,39 +134,13 @@ export const action: ActionFunction = async ({ request }) => {
...
-
-export default function ProjectStatusDataGrid() {
+function ProjectStatusDataGrid() {
   const fetcher = useFetcher();
   const { statuses } = useLoaderData() as LoaderData;
   const createButtonText = "Create New Status";
   const [openDeleteModal, setOpenDeleteModal] = useState<boolean>(false);
+  const [openEditModal, setOpenEditModal] = useState<boolean>(false);
+  const [openCreateModal, setOpenCreateModal] = useState<boolean>(false);
   const [error, setError] = useState<string>("");
```
Comment (JCoronaWize) [inline app/routes/manager/filter-tags/statuses.tsx:144]: we could remove this "error, setError" useState if we are not going to use it to trigger any action or display a message.

File: app/models/status.server.ts [hunk 3]
```diff
@@ -57,6 +57,7 @@ export async function updateProjectStatus({ id, name, stage }: updateProjectStat
   if (stage === "none") {
     data.stage = null
   }
-
+  // eslint-disable-next-line no-console
+  console.log(data)
```
Comment (JCoronaWize) [inline app/models/status.server.ts:61]: Can we remove this console.log

File: app/routes/manager/filter-tags/innovation-tiers.tsx [hunk 4]
```diff
@@ -59,28 +50,25 @@ type InnovationTierItem = Awaited<
...
+    requisites: z.string().min(1, { message: "Requisites is required"}),
   })
 );
 
-const validatorBack = withZod(
+const validatorForm = withZod(
   zfd.formData({
     name: z.object({ name: z.string() }).optional(),
-    ids: z.array(z.union([z.string(), z.number()])),
+
```
[the remark this answers is not in the document]
Comment (martinrobled0) [reply app/routes/manager/filter-tags/innovation-tiers.tsx:71]: you're right , I'll fix this

**Comment by martinrobled0** [comment]
in the #170 I was updating some modules (vm2 and vite) and seems that will affect the test (I pushed that changes here 😅 but I'll revert it) @juanan150

**Pull request merged**